modules/integration/tests-common/integration-common/src/main/java/org/wso2/iot/integration/common/extensions/AnalyticsServerExtension.java

```diff
@@ -63,8 +63,6 @@ public void onExecutionStart() throws AutomationFrameworkException {
         try {
             if (executionEnvironment.equalsIgnoreCase(ExecutionEnvironment.STANDALONE.name())) {
                 String carbonHome = serverManager.startServer("analytics");
-                log.info(carbonHome);
-                System.setProperty(ExtensionConstants.CARBON_HOME, carbonHome);
```

Comment:
I think IOT root directory should be considered as carbon home. Here since, we are starting analytics server at last it will consider {IOT_HOME}/wso2/analytics/ as the carbon home. This is required for the mobile qsg test cases. In https://github.com/wso2/product-iots/pull/453/files#diff-1f4c7a7b8befdfd60ad2f029639d6832R56, when we are getting carbon home, it return analytics home, but in order to find out the path for the mobile-qsg scripts, we need the path for the root folder.